### PR TITLE
Add dev env values and README example deployment cmds for argocd

### PR DESCRIPTION
**Pull request by brews**
Add helm chart values yaml file for a hypothetical `dev` environment.

Also add example `argocd` commands to deploy the `dev` values app into its own k8s Namespace, tracking development on the `main` branch.

### CODE REVIEW

File: daskhub-rhg/values-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Helm chart values for the prod environment
```
Comment (bolliger32) [inline daskhub-rhg/values-dev.yaml:1]: should we separate out users into its own yml file (or part of values.yml?) Seems like these should probably be the same across dev and prod? Or do you see reasons for keeping them differentiable across dev and prod

File: README.md [hunk 2]
```diff
@@ -1,25 +1,27 @@
 # daskhub-rhg-config
-Daskhub-flavored Jupyterhub deployment configuration.
+Daskhub-flavored Jupyterhub deployment configuration for argocd.
 
 This is a demonstration sandbox.
 
-
+## Fresh deployment
```
Comment (bolliger32) [inline README.md:6]: Will this production "fresh deployment" snippet need to be invoked manually after each tagged version? Or will we have a different argocd command than "app create" which says "modify the existing app to point to a new tag". Might be useful to put that info in the readme

**Review by bolliger32 — CHANGES_REQUESTED** [review]
@brews 2 quick comments - one about how we maintain the production argocd app after creating it, the other just wondering if we want different users across Dev and prod or if we can just assume those are always harmonized

File: daskhub-rhg/values-dev.yaml (hunk 1 above)
Comment (brews) [reply daskhub-rhg/values-dev.yaml:1]: If they're shared then could we put them into `values.yaml`?

For every new "values" file we add for the helm chart, we need to add another line when we create the app in `argocd`.

Comment (bolliger32) [reply daskhub-rhg/values-dev.yaml:1]: yeah I'm fine with putting them in values.yaml. I'm also fine with having one extra line that is --values users.yaml and maintaining an extra file. I kind of think of the users definition as separate than the other config stuff so made some organizational sense to me to maintain a separate file. But doesn't really matter in the end and I'm fine w/ whatever you think makes sense

File: README.md (hunk 2 above)
Comment (brews) [reply README.md:6]: Good idea! I added a bit on that in `26a61f94abb8c8c8461ed0e1e964bc4b8bd44435`.

File: daskhub-rhg/values-dev.yaml (hunk 1 above)
Comment (brews) [reply daskhub-rhg/values-dev.yaml:1]: Okay. I broke users back out into their own values file in `642ee95b934d1cf6dca88b9268a1cda0a544f099`. Also updated the README to add the extra values line.

**Review by bolliger32 — APPROVED** [review]
LGTM!

**Pull request merged**